utilities/generate_test_locales/justfile

```diff
@@ -1,5 +1,5 @@
 install:
-    pipenv install
+    ../../ov pdm install
 
 run:
-    pipenv run python .
+    ../../ov pdm run __main__.py
```

Comment:
We shouldn't use `ov` inside the just recipes, because it will cause `ov` recursion (:scream:), which _might_ be fine but is definitely not intentional...

These recipes should stay as-is and instead by called with `ov`, where `pdm` will be available to the recipes because they're then running inside the container.

Reply:
Fixed in [1c7d453](https://github.com/WordPress/openverse/pull/4626/commits/1c7d453f3c949c8ec169afea04ce9906b4da55db)